src/05-derivatives/03-imaging.md

```diff
@@ -153,9 +156,25 @@ manual_masks/
             sub-001_desc-tumor_mask.json
 ```
 
-## Segmentations and parcellations
+## Segmentations
 
-Common JSON metadata fields:
+A *segmentation* is a labeling of a spatial image with anatomical structures
+such that each location (for example, a voxel or a surface vertex) is identified
+with a label or a combination of labels.
+A *discrete segmentation* represents each structure with a unique integer
+label.
+A *probabilistic segmentation* represents each structure as values between
+0 and 1 (inclusive) at each location in the image, and one volume/frame per
+structure may be concatenated in a single file.
```

Comment:
Sorry, I'm gonna poke at this again :grimacing: 

This is technically another location where there are two different concepts getting mushed together (yes, that is the technical term):

- How are multiple segmented structures represented in a single file?
    - Within a single volume
    - Across volumes
- What data type is used to represent the segmentation?
    - Binary
    - Floating-point

For continuous / "probabilistic" data, storing multiple structures in a single volume is impossible, and so necessitates storing them across volumes.

For discrete data, we're accustomed to such being represented in a single volume / frame, using unique integer labels. This however enforces a one-to-one mapping between "elements" (e.g. voxels, vertices) and labels. In some instances (TractSeg comes to mind, as well as internal developments here), it's possible for one voxel to be included in multiple binary segmentations, at which point using unique integer labels does not work and one would need a 4D image. 

(In a way this could be thought of as a 4D `_mask.nii[.gz]` image as opposed to / in addition to a `_dseg` image...)

I don't actually have a fixed opinion on what to do here (if anything); just seeing if the spec can be made to cover the fundamentals, rather than unnecessarily excluding anything that strays slightly from the current most common use cases.